EIPS/eip-5334.md

```diff
@@ -0,0 +1,270 @@
+---
+eip: 5334
+title: 721 User And Expires And Level Extension
+description: Add a time-limited role with restricted permissions to ERC-721 tokens.
+author: Yan (@yan253319066)
+discussions-to: https://ethereum-magicians.org/t/erc-721-user-and-expires-and-level-extension/10097
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-25
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [EIP-721](./eip-721.md). It proposes an additional role (`user`) which can be granted to addresses, and a time where the role is automatically revoked (`expires`) and (`level`) . The `user` role represents permission to "use" the NFT, but not the ability to transfer it or set users.
```

Comment:
EIPs don't propose, they assert.  Also, saying "this standard" is a bit redundant.
```suggestion
An [EIP-721](./eip-721.md) extension that adds an additional role (`user`) which can be granted to addresses, and a time where the role is automatically revoked (`expires`) and (`level`) . The `user` role represents permission to "use" the NFT, but not the ability to transfer it or set users.
```